cellprofiler/modules/trackobjects.py

```diff
@@ -712,8 +734,50 @@ def create_settings(self):
             </ul></dd>
             </dl>''' % globals())
 
+        self.average_cell_diameter = cps.Float(
+            "Average cell diameter in pixels",
+            35.0, minval=5, doc='''\
+            %(ONLY_IF_FOLLOWNEIGHBORS)s<br>
+            The average cell diameter is used to scale many %(TM_FOLLOWNEIGHBORS)s algorithm parameters. 
+            Please use e.g. ImageJ to measure the average cell diameter in pixels.
+            ''' % globals()
+        )
+
+        self.advanced_parameters = cps.Binary(
+            'Use advanced configuration parameters', False, doc="""
+            %(ONLY_IF_FOLLOWNEIGHBORS)s<br>
+            Do you want to use advanced parameters to configure plugin? They allow for more flexibility,
+            however usually the default parameters suffice.
+            """ % globals()
+        )
+
+        self.drop_cost = cps.Float(
+            "Cost of cell to empty matching",
+            15, minval=1, maxval=200, doc='''\
+            %(ONLY_IF_FOLLOWNEIGHBORS)s<br>
+            The cost of considering cell (from frame t) not present in frame t+1. Increasing this value leads to more 
+            cells (from t) being matched with cells (from t+1) rather then classified as missing.
+            <dl>
+            <dd><img src="memory:%(PROTIP_RECOMEND_ICON)s">&nbsp; Recommendations:        
+            <ul>
+            <li>Too high value might cause incorrect cells to match between the frames. </li>
+            <li>Too low value might make the algorithm not to match cells between the frames.</li>
+            </ul></dd>
+            </dl>
+            ''' % globals()
+        )
+
+        self.area_weight = cps.Float(
+            "Weight of area difference in function matching cost",
+            25, minval=1, doc='''\
+            %(ONLY_IF_FOLLOWNEIGHBORS)s<br>
+            Increasing this value will make area difference more important than cells positions while 
```

Comment:
When it is increase it will favor diff in position over dif in area.
In other words it will prefer to match different position than different area.